salestax.rb

```diff
@@ -0,0 +1,52 @@
+input = []
+ans = 'y'
+non_taxable = ["book", "chocolate","chocolates", "pills"]
+
+while ans != 'n' do
+  i = gets()
+  input.push(i)
+  puts "more items in basket?"
+  ans = gets()
+  ans = ans.chomp
+end 
+
+total = 0
+tax = 0
+flag = 0
+
+input.each do |i|
+  flag = 0
+  item_tax = 0
+  output = i.split()	
+  qty = (output[0]).chomp.to_i
+  amt = (output[-1]).chomp.to_f
+  item_total = qty*amt
+  if qty <= 0 || amt <=0.0
+    puts "invalid input"
+    break
+  end
+  item_tax += item_total*0.05 if output.include?("imported")
+  #puts " item tax1: #{item_tax}"
+  output.each do |o|
+    #o == "at" ? (print ":") : (print "#{o} ")
```

Comment:
What's this for?

Reply:
To check if an item is imported, so if the keyword imported is present then additional tax should be added